orchestrator/schedules/scheduler.py

```diff
@@ -29,8 +30,15 @@
 from orchestrator.utils.helpers import camel_to_snake, to_camel
 
 jobstores = {"default": SQLAlchemyJobStore(url=str(app_settings.DATABASE_URI))}
+executors = {
+    "default": ThreadPoolExecutor(1),
+    "processpool": ProcessPoolExecutor(1),
```

Comment:
Is this used anywhere?

Reply:
yea processpool isn't necessary, thought I already removed it.
```suggestion
```